tests/unit/contrib/test_dpuczdx8g.py

```diff
@@ -270,6 +271,24 @@ def test_compile_conv2d_pool2d(self):
             ],
         )
 
+    @unittest.skipIf(not is_dpuczdx8g_vart_flow_enabled(), "DPUCZDX8G VART test")
+    def test_pad_conv2d_pool2d_partition(self):
+        partition_pad_conv2d_pool2d_nhwc_oihw_test(
+            (1, 10, 10, 1),
+            [[0, 0], [2, 2], [2, 2], [0, 0]],
+            1,
+            (2, 1, 2, 2),
+            [1, 1],
+            [1, 1],
+            [4, 4],
+            "Max",
+            [2, 2],
+            [0, 0],
+            targets=[
+                "DPUCZDX8G-zcu104",
```

Comment:
Can you add more targets to this test?

Reply:
And we should also add tests for the other DPUs

Reply:
Added test cases for other dpus as well.